longformer/longformer.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, config, layer_id):
         assert self.attention_dilation > 0
         self.autoregressive = config.autoregressive
 
-    def forward(self, hidden_states, attention_mask=None, head_mask=None):
+    def forward(self, hidden_states, attention_mask=None, head_mask=None, *args, **kwargs):
```

Comment:
Keyword arguments must appear after positional arguments and there are already keyword arguments in this function declaration, so `*args` needs to be removed.

Reply:
Actually, that wouldn't help - the usage of this component is here: https://github.com/huggingface/transformers/blob/master/src/transformers/modeling_bert.py#L314
and looks like the following:
```
self_outputs = self.self(
    hidden_states, attention_mask, head_mask, encoder_hidden_states, encoder_attention_mask
)
```
so `**kwargs` alone would not help. Perhaps it's better if I just copy the argument names from Transformers directly to avoid the confusion.

Reply:
BTW I got what you meant - it seems `*args` is a special case and it can actually be put after keyword arguments (at least in Python 3.8):
```
In [1]: def f(x, y=None, *args, **kwargs):
   ...:     print(x, y, args, kwargs)
   ...:

In [2]: f(1)
1 None () {}
```